src/RuleSet.php

```diff
@@ -109,7 +109,18 @@
             'normalize_index_brace' => true,
             'object_operator_without_whitespace' => true,
             'php_unit_fqcn_annotation' => true,
-            'phpdoc_align' => true,
+            'phpdoc_align' => [
+                // @TODO: on 3.0 switch whole rule to `=> true`, currently we use custom config that will be default on 3.0
+                'tags' => [
+                    'method',
+                    'param',
+                    'property',
+                    'return',
+                    'throws',
+                    'type',
+                    'var',
+                ],
```

Comment:
Sf is following this new feature, which is disabled by default on 2.x and will be enabled by default on 3.x